snuba/manual_jobs/job_loader.py

```diff
@@ -1,15 +1,15 @@
-from typing import Any, cast
+from typing import cast
 
-from snuba.manual_jobs import Job
+from snuba.manual_jobs import Job, JobSpec
 
 
 class JobLoader:
     @staticmethod
-    def get_job_instance(class_name: str, dry_run: bool, **kwargs: Any) -> "Job":
-        job_type_class = Job.class_from_name(class_name)
+    def get_job_instance(job_spec: JobSpec, dry_run: bool) -> "Job":
+        job_type_class = Job.class_from_name(job_spec.job_type)
         if job_type_class is None:
             raise Exception(
-                f"Job does not exist. Did you make a file {class_name}.py yet?"
+                f"Job does not exist. Did you make a file {job_spec.job_type}.py yet?"
```

Comment:
Does it make sense to create a custom exception instead of the base `Exception`?